views/index.html

```diff
@@ -16,15 +16,11 @@
     <script src="/script.js" defer></script>
   </head>
   <body>
-    <header>
-      <h1>HTML-only chat</h1>
-    </header>
-
-    <main>
-      <iframe src="/messages"></iframe>
-      <form action="/messages" method="post">
-        <input name="message" type="text" />
-        <button type="submit">submit</button>
+    <main class="c-chat">
+      <iframe class="c-chat__view" src="/messages#latest"></iframe>
+      <form class="c-chat__chat-form" action="/messages" method="post" autocomplete="off">
```

Comment:
Killed autocomplete; I don't think there is much point for such a light implementation remembering recent messages.

Reply:
Perfect, that's great.